python/lib/dcoscli/dcoscli/marathon/main.py

```diff
@@ -488,10 +488,6 @@ def group_add(self, group_resource, id_):
         else:
             group_resource = self._resource_reader.get_resource(group_resource)
 
-        if 'enforceRole' not in group_resource:
-            # enforceRole by default, this is in-line with the UI.
-            group_resource['enforceRole'] = True
```

Comment:
This was added for consistency with the UI, they are setting it to true by default when creating a group. However I had missed that it's only applicable to top-level groups. It's still something we should do for top-level groups.

Reply:
However the behavior changed recently in DC/OS (https://jira.mesosphere.com/browse/DCOS_OSS-5532). So passing it explicitly might not be needed anymore for top-level groups, I'll check it out tomorrow on a dev DC/OS version.

Reply:
Can you link to UI code and JIRA issue?
I think we should not apply our custom defaults. If this is a "good" default value then Marathon  should apply it. Imagine user who try to deploy application and try to debug why JSON was altered. There is no information in changelog about this change, nor test. We should remove this code.

cc. @meichstedt

Reply:
> Can you link to UI code and JIRA issue?
I think we should not apply our custom defaults. If this is a "good" default value then Marathon should apply it. 

See the issue I've pasted above, it is exactly about that. I've just tested against a 2.1-dev DC/OS cluster:

> dcos http -X POST -d '{"id": "/my-group"}' /service/marathon/v2/groups

And the group has `enforceRole` set to true by default.

```
  {
    "apps": [],
    "dependencies": [],
    "enforceRole": true,
    "groups": [],
    "id": "/my-group",
    "pods": [],
    "version": "2019-10-04T09:59:16.595Z"
  }
```

So we don't have to pass it explicitly anymore.